### ISSUE TITLE
Model: convert output probabilities to logit scores that favors being bona fide

### ISSUE THREAD

**Issue by TonyWangX** [Yaselley/deepfense-framework#2]
Current code uses `probs` to compute the metrics `results["average"] = self._compute_metrics(labels, probs)`.
https://github.com/Yaselley/deepfense-framework/blob/6a5e6f0bcf74ebcf1f07ec0d4f87ade8dff33941/deepfense/training/standard_trainer.py#L192

This does not work because
1. shape mismatch: `probs` is a matrix in shape `[N, 2]`, but all the metrics in `evaluations` requires a score in shape `[N]` -- only a single score per file.
2. `probs` is not the `score` needed by the metrics in `evaluations`. All the metrics require, ideally, log-likelihood-ratios. (For example, it doesn't make sense to compute Cllr or actDCF given probabilities.) 

Suggested change around this line 
https://github.com/Yaselley/deepfense-framework/blob/6a5e6f0bcf74ebcf1f07ec0d4f87ade8dff33941/deepfense/models/detector.py#L66
1. Adding a function to convert probs to proper scores inside detector, and return them. Suppose `outputs[:, 1]` correspond to the logits for bona fide. `scores = outputs[:, 1] - outputs[:, 0]`, this is the score we need. 
2. Feed the scores to compute_metrics.

Note: all metrics assume that a higher score indicates a higher chance of target (positive) class. Since all metrics treat `bonafide` as target_scores,  we need to make sure a higher score corresponds to a higher chance of being bona fide.
